### PR TITLE
Events

### PR DESCRIPTION
**Pull request by luccamicale**
I create the events inside my project to fiinish and complete. The project is great and should be approved!!

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,32 +1,52 @@
...
+          <button onClick={this.onClickHandler} type="button" id="divide" name="÷">÷</button>
+          <button onClick={this.onClickHandler} type="button" name="7">7</button>
+          <button onClick={this.onClickHandler} type="button" name="8">8</button>
+          <button onClick={this.onClickHandler} type="button" name="9">9</button>
+          <button onClick={this.onClickHandler} type="button" id="mult" name="*">x</button>
+          <button onClick={this.onClickHandler} type="button" name="4">4</button>
+          <button onClick={this.onClickHandler} type="button" name="5">5</button>
+          <button onClick={this.onClickHandler} type="button" name="6">6</button>
+          <button onClick={this.onClickHandler} type="button" id="less">-;</button>
+          <button onClick={this.onClickHandler} type="button" name="1">1</button>
```
Comment (Ugboaja-Uchechi) [inline src/components/calculator.js]: - [x] Great job with your project 👏 . Unfortunately, when I click the subtract button it doesn't work. Kindly fix this 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @luccamicale, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luccamicale** [comment]
I fixed the problem with the button '-'. Now my project is complete and it should be approved!!!!

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @luccamicale  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾
- Linter checks passing ✔️  
- Calculator working as intended 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**